sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/validator/AADJwtIssuerValidator.java

```diff
@@ -14,28 +14,14 @@
  */
 public class AADJwtIssuerValidator implements OAuth2TokenValidator<Jwt> {
 
-    private static final String LOGIN_MICROSOFT_ONLINE_ISSUER = "https://login.microsoftonline.com/";
-    private static final String STS_WINDOWS_ISSUER = "https://sts.windows.net/";
-    private static final String STS_CHINA_CLOUD_API_ISSUER = "https://sts.chinacloudapi.cn/";
     private final AADJwtClaimValidator<String> validator;
 
     /**
      * Constructs a {@link AADJwtIssuerValidator} using the provided parameters
      */
     @SuppressWarnings({"unchecked", "rawtypes"})
-    public AADJwtIssuerValidator() {
-        this.validator = new AADJwtClaimValidator(AADTokenClaim.ISS, validIssuer());
-    }
-
-    private Predicate<String> validIssuer() {
-        return iss -> {
-            if (iss == null) {
-                return false;
-            }
-            return iss.startsWith(LOGIN_MICROSOFT_ONLINE_ISSUER)
-                || iss.startsWith(STS_WINDOWS_ISSUER)
-                || iss.startsWith(STS_CHINA_CLOUD_API_ISSUER);
-        };
+    public AADJwtIssuerValidator(List<String> trustedIssuers) {
```

Comment:
This change is not correct, if your resource server is a multi-tenant application, then the `iss` could be any tenant?